src/TestSuite/MiddlewareDispatcher.php

```diff
@@ -48,7 +48,7 @@ class MiddlewareDispatcher
      * The application class name
      *
      * @var string
-     * @psalm-var class-string
+     * @psalm-var \Cake\Core\HttpApplicationInterface::class
```

Comment:
```suggestion
     * @psalm-var class-string<\Cake\Core\HttpApplicationInterface>
```
This is the syntax that other parsers have adopted.